src/Entity/Article.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\ArticleRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+
+/**
+ * @ORM\Entity(repositoryClass=ArticleRepository::class)
+ */
+class Article
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=255, nullable=true)
```

Comment:
sorry for the nullable, it seems to be not inlined with the Article form validation.
does this field ok to be nullable or not?

Reply:
Yes, nullable should be false. Thanks for pointing it out.